src/components/Header.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-    <v-toolbar app absolute ref="header">
+    <v-toolbar app absolute ref="header" v-if="saved">
```

Comment:
```
<v-expand-transition>
```

This should be added (don't forget to close it after </v-toolbar> to add a small transition